### ISSUE TITLE
feat: `client.logout()`

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#2400]

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}
//set the version type
process.env.VERS = "patch"

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci $VERS`);

// types only package
exec('npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN')
exec(`cd ./types-only && npm version $VERS && npm run build && npm publish && cd ..`);
exec(`git commit -a -m 'updated types-only package'`);
exec(`git push --force`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.28.6](https://github.com/open-wa/wa-automate-nodejs/compare/4.28.5...4.28.6) (2022-01-04)

- Release 4.28.6 [`37ccc34`](https://github.com/open-wa/wa-automate-nodejs/commit/37ccc34bf4adad04fbbf2fb88d34de98b00ff8f0)
- ✨ feat: `client.logout()` `POST /logout` #2400 [`f78c9c0`](https://github.com/open-wa/wa-automate-nodejs/commit/f78c9c081903904100028052090291c62b5b2bf8)
- 🥚 patches update [`4bf7a72`](https://github.com/open-wa/wa-automate-nodejs/commit/4bf7a72d291e282649ace52bc44fbc0fd8d342fa)
- 🥚 patches update [`bbd0c78`](https://github.com/open-wa/wa-automate-nodejs/commit/bbd0c78cda8e9b720e9b9968f45adc7886d41317)
- ✏️ fix typo in `qucik run` [h/t SC] [`78212a5`](https://github.com/open-wa/wa-automate-nodejs/commit/78212a5e1e2c65002429530dd94ebf91834dd534)
- updated types-only package [`4cf80ac`](https://github.com/open-wa/wa-automate-nodejs/commit/4cf80aca39ed61f84c03e4e4122abd658865d22a)
